### PR TITLE
[CPU] PagedAttention support u8 kvcache 

### PR DESCRIPTION
**Pull request by luo-cheng2021**
Support u8 kvcache to save some memory access time.

### CODE REVIEW

File: vllm/config.py [hunk 1]
```diff
@@ -309,13 +309,15 @@ def __init__(
         cache_dtype: str,
         sliding_window: Optional[int] = None,
         enable_prefix_caching: bool = False,
+        is_openvino_cpu: bool = False
```
Comment (ilya-lavrenov) [inline vllm/config.py:312]: I suppose it's not allowed to change public vLLM classes.
can we handle it internally in OpenVINO entities?

Comment (luo-cheng2021) [reply vllm/config.py]: Removed changes in public vLLM classes, use environment var `VLLM_OPENVINO_CPU_KV_CACHE_PRECISION=u8` to enable it.

File: vllm/executor/openvino_executor.py [hunk 2]
```diff
@@ -455,6 +462,8 @@ def __init__(
         device_config: DeviceConfig,
         lora_config: Optional[LoRAConfig],
     ) -> None:
+        if os.environ.get("VLLM_OPENVINO_CPU_KV_CACHE_PRECISION", "") == "u8":
```
[the remark this answers is not in the document]
Comment (luo-cheng2021) [reply vllm/executor/openvino_executor.py:465]: Added.

**Review by ilya-lavrenov — APPROVED** [review]

File: vllm/executor/openvino_executor.py [hunk 3]
```diff
@@ -185,6 +189,9 @@ def get_cache_block_size(
         device_config: DeviceConfig,
     ) -> int:
         head_size = model_config.get_head_size()
+        if device_config.device.type == "cpu":
+            if cache_dtype == "u8":
+                head_size += 8
```
Comment (dmitry-gorokhov) [inline vllm/executor/openvino_executor.py:194]: @luo-cheng2021  Could you please add a comment in separate PR why we need this addition. Worth to describe that quantization params are inlined in the cache.

Comment (luo-cheng2021) [reply vllm/executor/openvino_executor.py:194]: Added in #32.

**Pull request merged**